packages/terra-date-picker/tests/wdio/date-picker-spec.js

```diff
@@ -457,19 +457,6 @@ Terra.describeViewports('Date Picker', ['medium'], () => {
     });
 
     Terra.it.matchesScreenshot('ordinal date not set');
-
-    it('Open the date picker', () => {
```

Comment:
Why are these tests removed? Did we need to add an additional scenario to validate this fix?

Reply:
When we open the calendar after clearing out the date, today's date will be in bold which will change evereyday so I removed the test.

Reply:
@saket2403 Can we update the test to validate this interaction without relying on a screenshot?

Reply:
I can use mockdate to fix the current date and then take a screenshot, something similar to what will be done in #1202 ?

Reply:
Do we need to take a screenshot to validate this is working correctly ?

Reply:
We do take a screenshot after clearing the value, should be enough I guess.